deploy/aws/Dockerfile-aws.deploy

```diff
@@ -80,5 +86,7 @@ COPY ./services/frontend/nginx/aws/simple/default.conf /etc/nginx/conf.d/default
 # expose port
 EXPOSE 80
 
-# run nginx
-CMD ["nginx", "-g", "daemon off;"]
+# run server and nginx using sh file
+COPY ./deploy/aws/entrypoint-simple.sh /usr/src/app
+RUN chmod +x /usr/src/app/entrypoint-simple.sh
```

Comment:
simple vs. not-simple should be two .deploy files?

Reply:
In not-simple version we build two images separately, so we do not need a Dockerfile for not-simple version in aws folder.